### ISSUE TITLE
[BUG] Fix dpkg error when installing dashboard on AWS

### ISSUE THREAD

**Issue by johnbradley** [NVIDIA/NVFlare#1831]
**Describe the bug**
When installing the dashboard I receive this error:
```
...
Installing docker engine in nvflare_dashboard, may take a few minutes.
.../nvflare/NVFlare/aws_start_dsb.sh: line 92: dpkg: command not found
.../nvflare/NVFlare/aws_start_dsb.sh: line 92: lsb_release: command not found
Warning: Permanently added '35.92.15.96' (ED25519) to the list of known hosts.
Connection to 35.92.15.96 closed.
installing docker engine failed
```

**To Reproduce**
1. `git clone https://github.com/NVIDIA/NVFlare.git`
2. `cd NVFlare`
3. `git switch main`
4. `pip install .`
5. `nvflare dashboard --cloud aws`
6. Enter email address in the prompt
7. See error

**Expected behavior**
The dashboard to install without error.

**Desktop (please complete the following information):**
 - OS: [e.g. ubuntu 16.04] macOS 12.6.6
 - Python Version [e.g. 3.8]: 3.8.5
 - NVFlare Version [e.g. 2.1]: 2.2.2+22.ge217679c

**Additional context**
It seems like the code has the region hard coded.
So I had to run the following to change my default AWS region:
```
export AWS_DEFAULT_REGION=us-west-2
```

**Comment by IsaacYangSLA** [comment]
@johnbradley , may I know if you are using the default AMI ami-04bad3c587fe60d89?  This image is ubuntu 20.04 server in us-west-2 (https://cloud-images.ubuntu.com/query/focal/server/released.txt).  The command errors in your report seemed caused by non-ubuntu AMI.

If you would like to run the dashboard in different AWS region, you can edit that aws_start_dsb.sh.  This shell script is basically a convenient way to run dashboard on the cloud and also tries to minimize user interaction.  If you plan to change the region, please check that released.txt file to see which AMI should be used.

**Comment by johnbradley** [comment]
@IsaacYangSLA I didn't intentionally change the AMI so I assume it was the default. Changing the `aws_start_dsb.sh` script sounds like my next step. Thanks.

**Comment by johnbradley** [comment]
The `aws_start_dsb.sh` script seems to only exist temporarily so I ran `nvflare dashboard --cloud aws` and pressed Ctrl-C to terminate. Then I was able to change the region and AMI for the region I wanted to use. I still had the errors related to `dpkg` and `lsb_release` even though the VM was running Ubuntu.
This is the command where the error came from:
https://github.com/NVIDIA/NVFlare/blob/666ca811205cffe8ead7a6deeafd49d690d02cef/nvflare/lighter/impl/master_template.yml#L1980-L1988
It looks like the `$(dpkg --print-architecture)` and `$(lsb_release -cs)` parts are running on my local machine(mac) instead of within the VM. I replaced these values with the results when running on the AWS VM and everything worked fine.
